lib/engine/game/base.rb

```diff
@@ -131,6 +131,58 @@ def self.title
         name.split('::').last.slice(1..-1)
       end
 
+      def self.load_from_json(json)
```

Comment:
i'm not sure if it's worth the complexity, but it would be theoretically possible to lead the json directly, if you want to do this instead of putting the json in a ruby file let me know

Reply:
Yep, later on let's do that whenever ready :)